results.tex

```diff
@@ -80,33 +82,35 @@ \subsection{Separations Outage: Fuel Cycles with Supply Disruption}
 \ref{fig:base}. The outage begins at $t_i = 250$, lasts 50 time steps, ending at
 $t_f = 300$. During the outage, the remaining facilities in the supply chain
 operate normally, and the flow of fuel into and out of reactors adapts according
-to the state of available fresh fuel.
+to the state of available fresh fuel. Importantly, neither the \separations or
+\fabrication facilities have throughput constraints, i.e., both facilities are
+immediately able to process any quantity of fuel.
```

Comment:
Should there be another sentence justifying why we don't impose such constraints here?  It's either irrelevant to the point we are making about the DRE, or including it would make it harder to make that point???